frontend/src/components/AboutMe/ProfileCard.jsx

```
@@ -101,13 +133,16 @@
-          onClick={triggerFileInput}
-          aria-label="Change profile picture" // Describes the action for screen readers
-        >
-          <Camera className="text-white w-6 h-6" />
-        </button>
+        {isOwner && (
+          <button
+            className="absolute inset-0 rounded-full bg-black bg-opacity-50 opacity-0 group-hover:opacity-100 flex items-center justify-center transition duration-300 cursor-pointer"
+            onClick={triggerFileInput}
+            aria-label="Change profile picture"
+          >
+            <Camera className="text-white w-6 h-6" />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)